packages/client/components/TimelineFeedList.tsx

```diff
@@ -28,7 +28,8 @@ const TimelineFeedList = (props: Props) => {
       fragment TimelineFeedList_query on Query
       @refetchable(queryName: "TimelineFeedListPaginationQuery") {
         viewer {
-          timeline(first: $first, after: $after) @connection(key: "TimelineFeedList_timeline") {
+          timeline(first: $first, after: $after, eventTypes: $eventTypes)
```

Comment:
-1 if there are no events for that eventType, there's an empty page. I think some empty state would be helpful here. Could add this in a separate PR 

![Screenshot 2023-07-03 at 16 39 12](https://github.com/ParabolInc/parabol/assets/39854876/3f1039c2-0c51-41a2-ba38-af4d3e1eadbe)

Reply:
There should always be account creation and default team creation events (you can't archive them as meeting events so they should always be there): 

<img width="669" alt="Screenshot 2023-07-08 at 14 22 31" src="https://github.com/ParabolInc/parabol/assets/1879975/b64bd2ac-d502-4597-bbcf-0c72cb4416e9">

The fact that you have nothing locally feels like a local data issue. I don't have tension changing this at the moment.

Reply:
> There should always be account creation and default team creation events

But if you filter by Sprint Poker, it would only show the Sprint Poker events? So if you don't have any Sprint Poker meetings, I think we need to show some empty state to avoid a blank page